joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/util/TypeInfoCalculator.scala

```diff
@@ -232,26 +230,21 @@ object TypeInfoCalculator {
   }
 
   object TypeConstants {
-    val Byte: String                = "byte"
-    val Short: String               = "short"
-    val Int: String                 = "int"
-    val Long: String                = "long"
-    val Float: String               = "float"
-    val Double: String              = "double"
-    val Char: String                = "char"
-    val Boolean: String             = "boolean"
-    val Object: String              = "java.lang.Object"
-    val Class: String               = "java.lang.Class"
-    val Iterator: String            = "java.util.Iterator"
-    val Void: String                = "void"
-    val UnresolvedType: String      = "<unresolvedType>"
-    val UnresolvedSignature: String = "<unresolvedSignature>"
-    val UnresolvedReceiver: String  = "<unresolvedReceiverType>"
+    val Byte: String           = "byte"
+    val Short: String          = "short"
+    val Int: String            = "int"
+    val Long: String           = "long"
+    val Float: String          = "float"
+    val Double: String         = "double"
+    val Char: String           = "char"
+    val Boolean: String        = "boolean"
+    val Object: String         = "java.lang.Object"
+    val Class: String          = "java.lang.Class"
+    val Iterator: String       = "java.util.Iterator"
+    val Void: String           = "void"
+    val UnresolvedType: String = "<unresolvedType>"
```

Comment:
I just remembered Kotlin has a name for unresolved types - can we use the same one here please?

https://github.com/joernio/joern/blob/c1a0cf499f4a9d5f670db746db8f903f5ece8c05/joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/types/Constants.scala#L5